README.md

```diff
@@ -1 +1,69 @@
-# Project-1-Setup-and-mobile-version-skeleton
\ No newline at end of file
+# Project-1-Setup-and-mobile-version-skeleton
+In this project, I set up a "Project-1-Setup-and-mobile-version-skeleton" repository. I create html, css, images files .I created a branche to be familiar with github , I created Pull request .
+The 2 sections I added to build for this project are:
+-the toolbar (or header):
+-the headline section with background image
+
+#Built With
+-HTML and CSS
+-Visual studio
+-Technologies used
+
+## Live Demo (if available)
+github repo link 
+[https://github.com/ManarOmari/Project-1-Setup-and-mobile-version-skeleton/pull/1]
+
+online version link 
+none 
+
+## Getting Started
+
++started to buils mobile sketon for a Protoflio template 
+
++first : header contains a logo and tool bar 
+
++then set background image for the content section which conains a text and images 
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
```

Comment:
- [ ] you did great using the correct readme template, now kindly try to fill the usage, install, and setup sections to guide anyone who want to run your project

Reply:
Updated the installation and setup